src/App/component/AddRepositoryDialog.jsx

```diff
@@ -28,6 +28,23 @@ export default function AddRepositoryDialog({ open, reloadProjects, handleClose,
 
   let repositoryURL = ""
 
+  const config = {
+    headers: {
+      ...(jwtToken && { "Authorization": jwtToken })
+    }
+  }
```

Comment:
A `config` here only includes `headers` may causes some misunderstanding with the type that `axios` has already declared.

Please see [HERE](https://github.com/axios/axios/blob/4fbf61dc751ad69c7df9977d6edf7cf7ee87a5fb/index.d.ts#L76) for more details.

```suggestion
  const headers = { ...(jwtToken && { "Authorization": jwtToken }) }
```